.pre-commit-config.yaml

```diff
@@ -14,13 +14,14 @@ repos:
       name: "Chart Test: Lint"
       language: docker_image
       # pass_filenames: false
-      types: ['file']
-      files: '^charts/.*(\.ya?ml|\.tpl|\.helmignore|NOTES.txt)'
-      entry: -u 0 quay.io/helmpack/chart-testing:v3.4.0 ct
+      types: ['directory']
+      files: '^(.git|charts)'
+      entry: -u 0 quay.io/helmpack/chart-testing:v3.7.1 ct
```

Comment:
`ct` (chart testing) needs to see the git files to calculate if the chart needs to be bumped up